core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2531,6 +2531,8 @@ dao.monitor.isInConflictWithSeedNode=Your local data is not in consensus with at
   Please resync the DAO state.
 dao.monitor.isInConflictWithNonSeedNode=One of your peers is not in consensus with the network but your node \
   is in sync with the seed nodes.
+dao.monitor.isDaoStateBlockChainNotConnecting=You dao state chain is not connecting with the new data. \
+  Please resync the DAO state.
```

Comment:
Should be `Your dao state ...` and is there a reason why you used first `dao` in lower case and afterwards `DAO` in uppercase?

Reply:
Fixed